guides/release-notes.md

```diff
@@ -9,6 +9,20 @@ Lots of our bugs come from users like you and you can help us improve the docs b
 
 ---
 
+
+### New API: `wix-members` and `wix-members-backend`
+
+`wix-members` , `wix-members-backend`
+
+The `wix-users` and `wix-users-backend` APIs have been deprecated. Most of their functionality has moved to the new [`wix-members`](https://www.wix.com/velo/reference/wix-members) and [`wix-members-backend`](https://www.wix.com/velo/reference/wix-members-backend) APIs. The `emailContact()` and `emailUser()` (now `emailMember()`) functions have moved to the respective `TriggeredEmails` APIs of [`wix-crm`](https://www.wix.com/velo/reference/wix-crm/triggeredemails-obj) and [`wix-crm-backend`](https://www.wix.com/velo/reference/wix-crm-backend/triggeredemails-obj). Functions that handle consent policy have moved to the [`ConsentPolicy`](https://www.wix.com/velo/reference/wix-window/consentpolicy) API in `wix-window`. Finally, the `Authentication` API in `wix-members-backend` has two new functions. [`changeLoginEmail()`](https://www.wix.com/velo/reference/wix-members-backend/authentication-obj/changeloginemail) changes a site member's login email address. [`sendSetPasswordEmail()`](https://www.wix.com/velo/reference/wix-members-backend/authentication-obj/sendsetpasswordemail) sends a site member an email with a link to set their password. (November 2, 2021)
```

Comment:
There are actually 2 other new functions not mentioned in Monday: makeProfilePrivate/Public. See here: https://www.wix.com/velo/forum/news-announcements/wix-members-new-wix-members-wix-members-backend-apis
(also we use numerals (2 not two) even under 10. Thanks.